lib/rspec/core/pending.rb

```diff
@@ -38,7 +38,7 @@ class PendingExampleFixedError < StandardError; end
       # @param message [String] optional message to add to the summary report.
       #
       # @example
-      #     describe "an example" do
+      #     describe "some behaviour" do
```

Comment:
But sometimes it can be a method name? So it is less a behavior?

Reply:
Maybe change it to `"SomeClass"` like in the other example?